docs/Manual/04 Fusing.md

```diff
@@ -117,70 +116,72 @@ for i in range(0, 16, 4):
+The order of the dimensions in the fused schedule is as follows: first the fusing dimension `f`, then the fused dimensions *s*, followed by the unfused dimensions of `schedule0`, `schedule1`, and so on.
 
 We can easily calculate the number of dimensions in the fused schedule. For example, if we fuse the first *s* dimensions of a *d0*-dimensional space `schedule0` and a *d1*-dimensional space `schedule1`, the fused iteration space will have *s* fused dimensions, *d0 + d1 - 2s* unfused dimensions, and the special fusing dimension `f`, for a total of *d0 + d1 - s + 1* dimensions.
 
-As before, the `fuse` operation uses padding to ensure that the fused iteration space is not jagged in any direction. For example, say that `schedule0` is 4-dimensional, `schedule1` is 3-dimensional, and we partially fuse their first 2 dimensions:
+The `fuse` operation uses padding to ensure that the fused iteration space is not jagged in any direction. For example, consider that we partially fuse the first 2 dimensions of `schedule0`, which is 4-dimensional, and `schedule1`, which is 3-dimensional:
 ```python
 schedule = acc.fuse((schedule0, schedule1), partial=2)
 f, i, j, k, l, m = schedule.get_indices()
 ```
-The first dimension is the fusing dimensions `f`, whose size is 2. Next come the fused dimensions `i` and `j`. Then, `k` and `l` are the two unfused dimensions from `schedule0` and `m` is the unfused dimension from `schedule1`. The slice (0, \*, \*, \*, \*, 0) contains a copy of `schedule0`, the slice (1, \*, \*, 0, 0, \*) contains a copy of `schedule1`, and the rest of `schedule` is padded with empty elements. Note that full fusing is a special case of partial fusing, where `s` is the larger of the dimensions of `schedule0` and `schedule1`.
+The first dimension is the fusing dimensions `f` of size 2. Next comes the fused dimensions `i` and `j`, followed by the unfused dimensions `k` and `l` from `schedule0` and `m` from `schedule1`. The slice (0, \*, \*, \*, \*, 0) contains a copy of `schedule0`, the slice (1, \*, \*, 0, 0, \*) contains a copy of `schedule1`, and the rest of `schedule` is padded with empty elements. Note that full fusing is a special case of partial fusing, where `s` is the larger of the dimensions of `schedule0` and `schedule1`.
 
 ### Constraint 2: the fusing dimension always precedes unfused dimensions
-Partial fusing introduces a second constraint on the fusing dimension. Namely, the fusing dimension must precede all of the unfused dimensions in the dimension order. This constraint also applies to dimensions that are derived from the fusing dimension and from the unfused dimensions via splitting.
+Another constraint introduced by partial fusing restricts that the fusing dimension must precede all of the unfused dimensions in its dimension order. This constraint applies to dimensions derived from the fusing dimension and the unfused dimensions via splitting.
```

Comment:
Another constraint ... is that the fusing dimension must ...

(replace "restricts" with "is")